apps/precinct-scanner/src/screens/InsertBallotScreen.tsx

```diff
@@ -9,14 +9,19 @@ import * as format from '../utils/format'
 interface Props {
   scannedBallotCount: number
   electionDefinition: ElectionDefinition
+  currentPrecinctId?: Precinct['id']
 }
 
 const InsertBallotScreen: React.FC<Props> = ({
   scannedBallotCount,
   electionDefinition,
+  currentPrecinctId,
 }) => {
   return (
-    <CenteredScreen electionDefinition={electionDefinition}>
+    <CenteredScreen
+      electionDefinition={electionDefinition}
+      currentPrecinctId={currentPrecinctId}
```

Comment:
As we talked about before FYI I might refactor this to put these two things and machineConfig in a context... in general I was trying out uses parameters more then a context in this app but I think when things are being passed as params to a layout component like this its probably better in a context.

Reply:
Agreed :+1: